packages/backend/src/middlewares/validator.ts

```diff
@@ -0,0 +1,18 @@
+// import { HTTP400Error } from '../utils/httpErrors'
```

Comment:
Not a middleware now, we can move it outside of this folder.
A good enhancement would be if we validate the whole request object (including `params` and `queryParams`). Can keep things as is for now and incrementally adopt new changes.